srtcore/core.cpp

```diff
@@ -9772,19 +9784,19 @@ int srt::CUDT::handleSocketPacketReception(const vector<CUnit*>& incoming, bool&
 
                 LOGC(qrlog.Error, log << CONID() <<
                         "SEQUENCE DISCREPANCY. BREAKING CONNECTION."
-                        " seq=" << rpkt.m_iSeqNo
-                        << " buffer=(" << m_iRcvLastSkipAck
-                        << ":" << m_iRcvCurrSeqNo                   // -1 = size to last index
-                        << "+" << CSeqNo::incseq(m_iRcvLastSkipAck, int(m_pRcvBuffer->capacity()) - 1)
-                        << "), " << (offset-avail_bufsize+1)
+                        " %" << rpkt.m_iSeqNo
+                        << " buffer=(%" << bufseq
```

Comment:
Might be, but I'd postpone it until the common receiver as things may change there a lot. In case of a common receiver there will be a different buffer where data come from.